Description: Skip factor until flint 2.6 is released and packaged for Debian.
 Upstream is using a git snapshot for now since flint 2.5.2 can't
 handle factoring large integers.
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2020-05-13

--- a/M2/Macaulay2/packages/Macaulay2Doc/functions/isPrime-doc.m2
+++ b/M2/Macaulay2/packages/Macaulay2Doc/functions/isPrime-doc.m2
@@ -32,7 +32,6 @@
    	    isPseudoprime n
             isPrime n
             n1 = nextPrime(n+1)
-            factor(n1^2*n)
         Text
             These functions handle numbers larger than this.  For example,
         Example
@@ -40,7 +39,6 @@
             isPseudoprime m
             isPrime m
             isPrime m^2
-            factor m^2
         Example
             ndigits = 30
             m = nextPrime(10^30)
@@ -50,9 +48,6 @@
             isPrime m1
             isPrime (m*m1)
             isPrime(m*m*m1*m1*m2^6)
-            elapsedTime facs = factor(m*m1)
-            facs = facs//toList/toList
-            assert(set facs === set {{m,1}, {m1,1}})
             m3 = nextPrime (m^3)
             elapsedTime isPrime m3
             elapsedTime isPseudoprime m3
@@ -91,12 +86,6 @@
 	  "Since ", TO "factor", " returns factors guaranteed only to be pseudoprimes, it
 	  may be useful to check their primality, as follows."
 	  },
-     EXAMPLE lines ///
-     f = factor 28752093487520394720397634653456
-     peek'_2 f
-     first \ toList f
-     isPrime \ oo
-     ///,
      "This function can be used also to determine whether an ideal in a polynomial ring is prime.",
      EXAMPLE lines ///
      R = QQ[a..d];
--- a/M2/Macaulay2/packages/Macaulay2Doc/functions/factor-doc.m2
+++ b/M2/Macaulay2/packages/Macaulay2Doc/functions/factor-doc.m2
@@ -35,7 +35,6 @@
 	  exponent is an integer.",
 	  },
      EXAMPLE {
-         "factor 124744878111332355674003415153753485211381849014286981744945",
          "y = (2^15-4)/(2^15-5)",
       	 "x = factor y",
       	 "value x"
--- a/M2/Macaulay2/tests/normal/timing-quotient.m2
+++ b/M2/Macaulay2/tests/normal/timing-quotient.m2
@@ -215,9 +215,3 @@
 -- after fix: .166 seconds
 assert(numgens tim#1 == 33)
 assert(tim#0 < .5 * standardSecond)
-
-P=QQ[x,y,z,MonomialOrder=>Lex];
-d=z^4+z^2*x*y^9+z*x^9*y+x^5*y^5;
-phi=map(P,P,matrix{{x^13*y^4,x^3*y,x^20*y^6*z}});
-tim = timing factor(phi(d));
-assert(tim#0 < .05 * standardSecond)
